router/core/executor.go

```diff
@@ -96,6 +97,10 @@ func (b *ExecutorConfigurationBuilder) Build(ctx context.Context, opts *Executor
 		options.ResolvableOptions.ApolloCompatibilityReplaceInvalidVarError = true
 	}
 
+	if opts.ApolloRouterCompatibilityFlags.SubrequestHTTPError.Enabled {
```

Comment:
Now we have two, should we add an enable all like we have for the other original compatibility flags?

Reply:
For the two current flags and the use case I don't think an enable all is necessary yet, you'd only want to enable these if you depend on them, to aid migration to more standards compliant code. There's a good chance you could migrate from Apollo Router to Cosmo without hitting either of these issues I think.